### PR TITLE
Fixed button text on YouTube and Twitch form

### LINKED ISSUE #3759: Button has incorrect text when validating channels
**What happened**:
In the process of signing up, on the YouTube and Twitch page, the button is incorrectly displaying "validate blog" as below:
image

**What should have happened**:
The button should say something along the lines of "validate channel(s)".

**How to reproduce it (as precise as possible)**:
In the signup process, go to the YouTube and Twitch form.

**Anything else we need to know?**:

**Environment**:

- OS: Windows
- Browser: Chrome

### PR DESCRIPTION
**Pull request by NeilAn99**
## Issue This PR Addresses
Fixes #3759 

## Type of Change

- [x] **Bugfix**: Change which fixes an issue
- [ ] **New Feature**: Change which adds functionality
- [ ] **Documentation Update**: Change which improves documentation
- [x] **UI**: Change which improves UI

## Description

Fixes button label to correctly describe the action. As seen in the screenshot:
image

## Steps to test the PR

Go to signup, go to YouTube/Twitch form to see the fix.

## Checklist

- [x] **Quality**: This PR builds and passes our npm test and works locally
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not (if applicable)
- [x] **Documentation**: This PR includes updated/added documentation to user exposed functionality or configuration variables are added/changed or an explanation of why it does not(if applicable)

### CODE REVIEW

**Comment by Genne23v** [comment]
I think it's better to add some padding to the button. Maybe make it longer to put the text in one line.

**Comment by NeilAn99** [comment]
> I think it's better to add some padding to the button. Maybe make it longer to put the text in one line.

Good idea, will do that.

**Comment by NeilAn99** [comment]
Added some padding as seen below, I decided not to put the text all in one line as it made the button too long.
image

**Review by humphd — CHANGES_REQUESTED** [review]
While the text you've used is more accurate, it's also extremely cramped within the button.  Can we use `Validate` instead?

**Comment by NeilAn99** [comment]
> While the text you've used is more accurate, it's also extremely cramped within the button. Can we use `Validate` instead?

Sure thing, any thoughts on changing the blog form validate button also to be just `Validate`? See below:
image

**Comment by cychu42** [comment]
Looks nice.
I wonder if it's better to have the `VALIDATE` button comform to how the `PRVIOUS` and `NEXT` button looks, in terms of size and padding.

**Comment by NeilAn99** [comment]
This is how the `Validate` button looks using the styling of the `previous` and `next` buttons:
image
Which one do we prefer?

**Review by cychu42 — COMMENTED** [review]
For this iteration, while I think uniformity is nicer, the `VALIDATE` button does have more left over width space than the other buttons. I prefer this, but I’m not hard set on it if other people disagree.

**Comment by Ririio** [comment]
What I think would be nice is to hide the "next" button and replace it with the "validate". When the URL is valid the button gets replaced with "next". 

I can't test the signup portion so I'm unsure if it is required to have the feed url to create an account.

**Comment by NeilAn99** [comment]
> What I think would be nice is to hide the "next" button and replace it with the "validate". When the URL is valid the button gets replaced with "next".
> 
> I can't test the signup portion so I'm unsure if it is required to have the feed url to create an account.

I think this should be for another issue. Currently, you cannot go to the next page without first validating the current form.

**Comment by Ririio** [comment]
That's what I meant actually. The 'Validate' shows up on the location of 'Next'. When the URL is valid, the 'Next' button shows up, otherwise 'Validate' stays

**Review by humphd — APPROVED** [review]
I agree that we should do this fix here, and if we want to make larger changes to the way the form works, do that work in a follow-up.

This looks like a good solution to me.

**Comment by humphd** [comment]
@NeilAn99 can you squash/rebase this please?

**Comment by humphd** [comment]
@NeilAn99 that's a merge of master into your branch.  Can you please do a proper rebase of your work on `master` instead?

**Comment by NeilAn99** [comment]
I think I have rebased on master. Did I mess something up?

**Comment by humphd** [comment]
You likely did a `git pull` to update, right?  That will `fetch` and `merge`:

Screen Shot 2022-11-17 at 7 28 21 PM

**Comment by NeilAn99** [comment]
I think I accidentally pressed the `Update Branch` button. The master branch on my forked repo seems to have the squashed commit but the issue-3759 branch has the `Merge branch 'master' into issue-3759` commit.

**Comment by IvaniGabrovsky** [comment]
@NeilAn99 I took a look and everything looks good.

**Comment by cychu42** [comment]
I tried to fetch the issue branch from your fork. It doesn't look like the code from commit `65c3ec4 Changed blog validation schema to match channel` is preserved.
image
You would want to recreate that commit’s code.
Also, it doesn't look like you squashed the commits in the issue branch, and it's different from what we see in the PR (commit `65c3ec4 `doesn't exist in this PR)
Therefor, I think you would want to recreate the code for commit` 65c3ec4`, squash the commits, and then push to update the PR.

**Comment by NeilAn99** [comment]
So I recreated the commit and tried squashing, but only the recreated commit is showing up:
image
Should I just push this commit?

**Comment by humphd** [comment]
Sure, if you have it down to 1 commit, push away.  You can always push again if it's not right.

**Review by cychu42 — DISMISSED** [review]

File: src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx [hunk 1]
```diff
@@ -77,7 +77,7 @@ const useStyles = makeStyles((theme: Theme) =>
       width: '70%',
       padding: '4px',
       alignSelf: 'center',
-      fontSize: '0.8em',
+      fontSize: '1.1em',
```
Comment (anshul137) [inline src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx:80]: ## Optimal font size
There are no exact rules for font sizing, but there are some generally good practices to have Font sizes around `1.6rem` to `1.8rem`.

I would really appreciate to see other views as well on this. 
@humphd 
Thanks

**Comment by NeilAn99** [comment]
> It looks like you are doing merge instead. Can you walk us through what you do to squash, rebase, and push? While in the issue branch, I usually do `git rebase main -i` to squash, `git rebase main` for rebase, and `git push origin issue-xxx --force` to force push the rebased/squashed commit.

First I checkout to the issue branch `git checkout issue-3759`. Then I do `git rebase master -i`. Next I squash the commits into one. I have not done `git push origin issue-xxx --force`, so I'll try that, thanks for the walkthrough.

**Comment by NeilAn99** [comment]
@cychu42 I think that worked! Thank you again, I was getting lost.

**Comment by cychu42** [comment]
Glad to help. At least the commits are down to two.

**Comment by NeilAn99** [comment]
> Glad to help. At least the commits are down to two. You likely didn't fully squash all the commits. When you squash them via `git rebase master -i`, you can leave first commit as pick and rest as squash to combine them all into one commit.

That's what I did, I don't know why I have another commit. Here's what I have when I do `git rebase master -i`:
image

When I do `git status`:
image

When I do `git commit --amend`:
image

Any ideas?

**Comment by Genne23v** [comment]
@NeilAn99 I fetched your branch and took a look. It seems that a commit is made to your master branch. So basically `rebase master` means you are rebasing your topic branch on updated master. Which mean that's already two commits from this repo's master. 

```
commit cd615604e83732a766905730bb3ccffab27468f7 (NeilAn/issue-3759)
Author: Neil An <nan1@myseneca.ca>
Date:   Fri Nov 18 12:12:04 2022 -0500

    Changed blog validation schema to match channel

    added padding to validate buttons

    updated buttons text and size

    Changed blog validation schema to match channel

commit 0f8df23d57c1a35cbc4c6ac0b7d307aa8aff63cb (NeilAn/master)
Author: Neil An <nan1@myseneca.ca>
Date:   Mon Nov 14 11:13:20 2022 -0500

   Fixed validate channel button text

    added padding to validate buttons

    updated buttons text and size
```

I think what you can do is `git reset HEAD^1` to your master and try rebase again. But be careful reset is a very powerful command, you need to make sure what you are going to do or save your current working folder in somewhere.

**Comment by NeilAn99** [comment]
@humphd I think I got it down to 1 commit, please check it out.

**Comment by humphd** [comment]
@NeilAn99 it looks to me like you've brought in a change from another branch here (see your changes).  Can you please revert that so it's only your changes?

I'd move the font-size discussion to another issue so we don't drag this one out.

File: src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx (hunk 1 above)
Comment (humphd) [reply src/web/app/src/components/SignUp/Forms/RSSFeeds.tsx:80]: Let's leave this for a follow-up, if you want to file an issue on it.

**Review by humphd — APPROVED** [review]

**Pull request merged**